apps/codeforafrica/src/lib/api.ghost.js

```diff
@@ -0,0 +1,101 @@
+import GhostContentAPI from "@tryghost/content-api";
+
+import convertToCamelCase from "@/codeforafrica/utils/camelcaseKeys";
+
+function ghostAPI() {
+  return new GhostContentAPI({
+    url: process.env.GHOST_URL,
+    key: process.env.GHOST_API_KEY,
+    version: process.env.GHOST_API_VERSION || "v5",
+  });
+}
+
+function transformGhostPost(post) {
```

Comment:
```suggestion
function transformPost(post) {
```
We know it's Ghost from file name.